frigg_worker/jobs.py

```diff
@@ -84,7 +84,12 @@ def succeeded(self):
     def settings(self):
         return build_settings(self.working_directory, self.docker)
 
+    def start_services(self):
+        for service in self.settings['services']:
+            self.docker.run("sudo service {0} start".format(service))
+
     def run_tests(self):
+        self.start_services()
```

Comment:
This needs to be moved below 
```python
if not self.clone_repo():
             return self.error('git clone', 'Access denied')
```
because the settings cannot be read before cloning